### PR TITLE
Fixed multi line in users list

### LINKED ISSUE #9644: User id column is too short on users page
The IDs break and overlap on current master:

screenshot_20180529_124255

Try with a user id like: `6a34a394-a9c6-4759-a89c-1e4e1cd48724`

@skjnldsv

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #9644

|Before|After|
|:--:|:--:|
|kazam_screenshot_00002|kazam_screenshot_00000|

### CODE REVIEW

File: settings/src/components/userList/userRow.vue [hunk 1]
```diff
@@ -39,7 +39,8 @@
 				 :srcset="generateAvatar(user.id, 64)+' 2x, '+generateAvatar(user.id, 128)+' 4x'"
 				 v-if="!loading.delete && !loading.disable">
 		</div>
-		<div class="name">{{user.id}}</div>
+		<!-- dirty hack to ellipsis on two lines -->
+		<div class="name">{{user.id.substr(0, 40)}}{{user.id.substr(0, 40).length===40?'...':''}}</div>
```
Comment (jancborchardt) [inline settings/src/components/userList/userRow.vue]: Use a real ellipsis "…" instead of 3 dots ;)

File: settings/css/settings.scss [hunk 2]
```diff
@@ -1463,10 +1463,25 @@ doesnotexist:-o-prefocus, .strengthify-wrapper {
...
+				&.displayName,
+				&.mailAddress {
+					> input {
+						text-overflow: ellipsis;
+					}
+				}
+				&.name,
+				&.storageLocation {
+					// better multi-line visual
+					line-height: 1em;
```
Comment (jancborchardt) [inline settings/css/settings.scss]: Seems a bit squished. The line-height should never be lower than 130% or so.

**Review by jancborchardt — COMMENTED** [review]
See some detail comments :)

(Regarding line-height: rather than squishing the text so much, we could also limit it to one line.)

**Review by jancborchardt — APPROVED** [review]
Looking good :)

**Comment by MorrisJobke** [comment]
It would be nice to have the full name as title to be able to get the full name displayed.

**Review by MorrisJobke — APPROVED** [review]
Works nicely - just had a little nitpick 👍

**Comment by skjnldsv** [comment]
@MorrisJobke Make sense!

**Comment by skjnldsv** [comment]
@MorrisJobke I changed the behavior, I wasn't happy.
On webkit it will ellipsis on second line and just hide after the second line on other browsers

**Comment by MorrisJobke** [comment]
> @MorrisJobke I changed the behavior, I wasn't happy.
> On webkit it will ellipsis on second line and just hide after the second line on other browsers

But there is still no `title` attribute set on that item. It shows the ellipsis but does not show a title on long hover. 😕

**Comment by skjnldsv** [comment]
@MorrisJobke the whole text is still selectable though :)

**Pull request merged**